tests/Functional/FunctionalTest.php

```diff
@@ -21,14 +21,22 @@
 use function implode;
 use function in_array;
 use function rtrim;
+use function setlocale;
 use function str_replace;
 use function strpos;
 use function trim;
 
+use const LC_ALL;
+
 class FunctionalTest extends TestCase
 {
     private const RENDER_DOCUMENT_FILES = ['main-directive'];
 
+    protected function setUp(): void
+    {
+        setlocale(LC_ALL, 'en_US.utf8');
```

Comment:
Did you run into an error that is based on the locale in GitHub Action?

Reply:
Yes, `grün` was normalized as `grn`

Reply:
maybe this indicates an issue in the tool, that should not rely on such locale...